tests/unit/h/views/admin/groups_test.py

```diff
@@ -249,6 +276,16 @@ def test_read_renders_form_if_group_has_no_creator(self, pyramid_request, group)
 
         assert response["form"] == self._expected_form(group)
 
+    def test_read_renders_form_if_group_has_no_organization(
+        self, pyramid_request, group
+    ):
+        group.organization = None
+        view = GroupEditViews(GroupContext(group), pyramid_request)
+
+        response = view.read()
+
+        assert response["form"]["organization"] is None
```

Comment:
This is the test for the original page load crash in https://github.com/hypothesis/h/issues/9009